src/util/hass-translation.ts

```diff
@@ -1,58 +1,63 @@
 import { translationMetadata } from "../resources/translations-metadata";
+import { fetchFrontendUserData } from "../data/frontend-user-data";
+import { HomeAssistant } from "../types";
 
-export function getActiveTranslation() {
+/**
+ * Search for a matching translation from most specific to general
+ */
+function findAvilableLanguage(language: string) {
   // Perform case-insenstive comparison since browser isn't required to
   // report languages with specific cases.
-  const lookup = {};
-  /* eslint-disable no-undef */
+  const avilableLanguages = {
+    "zh-cn": "zh-Hans",
```

Comment:
What does this code do, it was not there before, is it some aliases ?

Reply:
Ah I see we had a variant before. Might want to add a comment what we're doing here.

Reply:
zh-MY and zh-MO are for Malaysia and Macau, they are not in ISO 639, but still has some usage

Reply:
Done